examples/data_preprocessing/police_killing/preprocessing_rules_from_paper.ipynb

```diff
@@ -0,0 +1,1009 @@
+       "      <th>label</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>[ ] TARGET / Chicago Tribune Lake County Major...</td>\n",
+       "      <td>0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>Round Lake police shooting Round Lake police s...</td>\n",
+       "      <td>0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>PERSON shooting PERSON shooting TARGET / Chica...</td>\n",
+       "      <td>0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>Scene of Round Lake police shooting Scene of R...</td>\n",
+       "      <td>0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>involved shooting TARGET / Chicago Tribune The...</td>\n",
+       "      <td>0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                             samples  label\n",
+       "0  [ ] TARGET / Chicago Tribune Lake County Major...      0\n",
+       "1  Round Lake police shooting Round Lake police s...      0\n",
+       "2  PERSON shooting PERSON shooting TARGET / Chica...      0\n",
+       "3  Scene of Round Lake police shooting Scene of R...      0\n",
+       "4  involved shooting TARGET / Chicago Tribune The...      0"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df_test.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting the rules"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Starting with simple rules based on the keywords of the paper.\n",
+    "\n",
+    "\"These lists were semi-automatically constructed by looking up the nearest neighbors of “police” and “kill” (by cosine distance) from Google’s public release of word2vec vectors pretrained on a very large (proprietary) Google News corpus,20 and then manually excluding a small number of misspelled words or redundant capitalizations (e.g. “Police” and “police”).\" (Keith et al. p. 11)\n",
+    "\n",
+    "I saved the keywords in a csv."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_keywords(data_path):\n",
+    "    keywords = pd.read_csv(os.path.join(data_path, \"keywords.csv\"))\n",
+    "    return(keywords)\n",
+    "\n",
+    "keywords = get_keywords(data_path)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#dictionary mapping the rules to their ID\n",
+    "\n",
+    "def rule2id(keywords):\n",
```

Comment:
bad function naming. Try to add some verb to the function name (e.g. "get", "calculate" etc)